ledger/complete/mtrie/forest_test.go

```diff
@@ -1038,37 +1038,6 @@ func TestValueSizesWithDuplicatedKeys(t *testing.T) {
 	}
 }
 
-func TestNow(t *testing.T) {
-
```

Comment:
do we know what was the original reason for this test ?

Reply:
Yes, `TestNow` was a throwaway test that wasn't meant to be pushed to the repo.  I use `TestNow` to verify my understanding locally and forgot to remove it in a prior PR.